.config/dotnet-tools.json

```diff
@@ -8,8 +8,8 @@
         "dotnet-gitversion"
       ]
     },
-    "octonotes.tool": {
-      "version": "0.1.171",
+    "octopus.octonotes.tool": {
```

Comment:
The package for our Octonotes tool got renamed, this fixes that up and updates to the latest version as part of a fix up for build issues, see https://octopusdeploy.slack.com/archives/C01HH8T16G3/p1712011616837369